### PR TITLE
ref(py3): Fix bigtable nodestore compression

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Fixes SENTRY-FOR-SENTRY-CD

### CODE REVIEW

File: tests/sentry/nodestore/bigtable/backend/tests.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 @pytest.mark.skip(reason="Bigtable is not available in CI")
 class BigtableNodeStorageTest(TestCase):
     def setUp(self):
-        self.ns = BigtableNodeStorage()
+        self.ns = BigtableNodeStorage(project="test")
```
Comment (EvanPurkhiser) [inline tests/sentry/nodestore/bigtable/backend/tests.py:13]: The project must be passed to the client to run these tests. Not sure if that's a new requirement, but it was not happy without it.

File: src/sentry/nodestore/bigtable/backend.py [hunk 2]
```diff
@@ -172,7 +172,7 @@ def set(self, id, data, ttl=None):
         self._set_cache_item(id, data)
 
     def encode_row(self, id, data, ttl=None):
-        data = json_dumps(data)
+        data = json_dumps(data).encode("utf-8")
```
Comment (wedamija) [inline src/sentry/nodestore/bigtable/backend.py:175]: Just want to confirm you tested on both py2 and py3?

Comment (wedamija) [reply src/sentry/nodestore/bigtable/backend.py:175]: @EvanPurkhiser verified this just now, everything looks good.

**Review by wedamija — APPROVED** [review]

File: src/sentry/nodestore/bigtable/backend.py (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/nodestore/bigtable/backend.py:175]: Verified this in py2, tests pass and I checked with unicode, works no problem.

**Pull request merged**